gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -1262,10 +1265,44 @@ def call_ttcp(self, other_args: List[str]):
             if not ns_parser:
                 return
 
+            exchange = "Uniswap"
```

Comment:
@DidierRLopes I am not sure if I implemented this difflib feature :D But would be great if you can take a look. I set exchange parameter to not required, and I removed `choices`.